XYZDrivers/src/main/java/com/xyzdrivers/repositories/MembersRepo.java

```diff
@@ -58,16 +74,22 @@ public Member get(Integer id) throws RepositoryException {
             results = this.sqlService.retrieve("MEMBERS", "*", keyColumn, keyValue);
             //parse members data
             for (Object[] memberData : results) {
+                String dateTime = df.format(memberData[3]);
+                String dateTime2 = df.format(memberData[4]);
+
+                dob.setTime(df.parse(dateTime));
+                dor.setTime(df.parse(dateTime2));
+
                 Member member = new Member(memberData[0].toString(), //id
                         memberData[1].toString(), //name
                         memberData[2].toString(), //address
-                        LocalDate.parse(memberData[3].toString()), //dob
-                        LocalDate.parse(memberData[4].toString()), //dor
+                                             dob, //dob
+                                             dor, //dor
                         memberData[5].toString(), //status
-                        (double) memberData[6]);                     //balance
+                        (double) memberData[6]);  //balance
                 members.add(member);
             }
-        } catch (SQLException | IllegalArgumentException ex) {
+        } catch (SQLException | IllegalArgumentException | ParseException ex) {
```

Comment:
There's no IllegalArgumentException to catch. You also don't check the arguments passed to this function